lessons/en/misc/plug.md

```diff
@@ -336,7 +347,17 @@ defmodule Example.Application do
     Supervisor.start_link(children, opts)
   end
 
+  # For Elixir < 1.9
   defp cowboy_port, do: Application.get_env(:example, :cowboy_port, 8080)
+  
+  # For Elixir 1.9+
+  defp cowboy_port do
+    case Application.fetch_env(:example, :cowboy_port) do
+      {:ok, value} -> value
+      :error -> 8080
+    end
+  end
```

Comment:
Perhaps I am missing something here, but is there something incorrect with using `Application.get_env/3`? If not, I think it may be confusing to indicate that it should not be used in Elixir 1.9+

Reply:
@kinson Apologies, this was a long time ago and I'm not sure what my rationale was at that point. I can't recall why I thought `fetch_env/2` would be better for 1.9+ than `get_env/3`. I'll change this back to use just `get_env/3`.